geode-core/src/integrationTest/java/org/apache/geode/management/internal/beans/DistributedSystemMBeanIntegrationTest.java

```diff
@@ -30,14 +31,16 @@
 import org.apache.geode.cache.RegionShortcut;
 import org.apache.geode.management.DistributedSystemMXBean;
 import org.apache.geode.management.internal.json.QueryResultFormatter;
+import org.apache.geode.management.model.Employee;
 import org.apache.geode.test.junit.assertions.TabularResultModelAssert;
 import org.apache.geode.test.junit.rules.GfshCommandRule;
 import org.apache.geode.test.junit.rules.MBeanServerConnectionRule;
 import org.apache.geode.test.junit.rules.ServerStarterRule;
 
 public class DistributedSystemMBeanIntegrationTest {
 
-  public static final String SELECT = "select * from /testRegion r where r.id=1";
+  public static final String SELECT_ALL = "select * from /testRegion r where r.id=1";
```

Comment:
Can we add a query that returns 0 results. And verify the results are shown as expected.

Reply:
the test is to verify the results are formatted correctly. If returning 0 results, no formats needed, that's not the purpose of the tests. I am sure there are other tests that tests 0 results.

Reply:
done